src/job/jobHandler.ts

```diff
@@ -286,8 +286,12 @@ export abstract class JobHandler {
     return '';
   }
 
-  protected constructManifestIndexPath(): Promise<void> {
-    return Promise.resolve();
+  // For certain unversioned properties, urlSlug is null; use branchName instead
+  protected constructManifestPrefix(): string {
```

Comment:
doing it here means it will hide derived class implementation, is this intended

Reply:
Also i believe manifest generation done only for prod jobs

Reply:
I figured since the manifest index path should be constructed in the same way by any derived class, it should be OK to move it to the parent class, where it is called from. I also put functionality to kick-off manifestJobs from githubPush just for now (in the other PR) with the thought that it might be easier for us to test. We can remove that functionality once we are confident with the search manifests results. Thoughts?